src/types/streams/streams_item.rs

```diff
@@ -0,0 +1,15 @@
+use crate::types::resource::Stream;
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use url::Url;
+
+#[serde_as]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct StreamsItem {
+    pub stream: Stream,
+    pub meta_id: String,
+    pub video_id: String,
+    pub transport_url: Url,
+    pub mtime: DateTime<Utc>,
```

Comment:
- [ ] Please add docs that this is the modify time and what it actually mean/when is updated.